### ISSUE TITLE
[SUGGESTION] Auto generate guis // auto sort cosmetics in guis

### ISSUE THREAD

**Issue by itsMarkiS** [HibiscusMC/HMCCosmetics#124]
### Feature Description

For example, addind a yml file with all of the cosmetics you could have an option to automatically generate a gui config for every single type of cosmetic, for example

"cosmetics.yml" contains back, offhand and hat cosmetics, and "cosm.yml" only have hats so it will generate a custom gui "cosmetics-back.yml" with all of the backpack cosmetics "cosmetics-offhand.yml" for the offhand cosmetics, "cosmetics-hat.yml" & "cosm-hat.yml" and etc

### Implementation Description

basically it will automatically generate a gui based in the type of the cosmetic

**Comment by LoJoSho** [comment]
This has been discussed a bit on the discord and my general consensus was to not add such a system because it conflicts with the current menu system. It becomes that everyone wishes to customize it when that wouldn't really be the case with menus that are auto generated. This would also be a partial duplicate of #91 and as such I am going to close this issue.
